src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -164,17 +192,11 @@ void AreaTrigger::Update(uint32 p_time)
     else
         UpdateSplinePosition(p_time);
 
-    sScriptMgr->OnAreaTriggerEntityUpdate(this, p_time);
-
-    if (GetDuration() != -1)
+    if (IsAIEnabled)
     {
-        if (GetDuration() > int32(p_time))
-            _UpdateDuration(_duration - p_time);
-        else
-        {
-            Remove(); // expired
-            return;
-        }
```

Comment:
I of course had a very good reason, i just can't remember it, so let's do like it never happened